lib/hooks.js

```diff
@@ -83,22 +109,19 @@ module.exports.useTrackPlayerProgress = (interval = 1000) => {
 
     const [state, setState] = useState(initialState);
 
-    async function getProgress() {
-        try {
-            const [position, bufferedPosition, duration] = await Promise.all([
-                TrackPlayer.getPosition(),
-                TrackPlayer.getBufferedPosition(),
-                TrackPlayer.getDuration()
-            ])
-
-            setState({ position, bufferedPosition, duration });
-        } catch (err) {
-            // The player is probably not ready yet.
-            // todo: better error handling
-        }
+    const getProgress = async () => {
```

Comment:
The try/catch block was taken from the progress component (https://github.com/react-native-kit/react-native-track-player/blob/0f411baf0c8b21184d4585c9515d805798b0a244/lib/ProgressComponent.js#L33-L46), I am not sure if we want to remove that error handling.

Reply:
This will only ever be called when playback state is one of `TrackPlayer.STATE_PLAYING` ir `TrackPlayer.STATE_BUFFERING`, which can only occur when the player is initialised

Reply:
Awesome, thanks for the clarification!